Esterni/VerbaliEsterni/2020_12_28/components/verbale.tex

```diff
@@ -0,0 +1,96 @@
+\section{Informazioni generali}
+
+\begin{itemize}
+
+	\item \textbf{Canale di comunicazione:} Zoom;
+	
+	\item \textbf{Data:} \DataMeeting{};
+	
+	\item \textbf{Ora inizio:} 16:20;
+	
+	\item \textbf{Ora fine:} 17:00;
+	
+	\item \textbf{Segretario:} \Lucrezia{};
+	
+	\item \textbf{Partecipanti}: \textit{Alessio Crestani}, \textit{Federico R.} come rappresentanti dell'azienda proponente \proponente{} e alcuni componenti del gruppo \Gruppo{}:
+	
+		\begin{itemize}
+			\item \Daniele{};
+			\item \Davide{};
+			\item \Francesco{};
+			\item \Giosue{};
+			\item \Lucrezia{};
+			\item \Matteo{}.
+		\end{itemize}
+
+
+
+	\item \textbf{Membri assenti}:
+		\begin{itemize}
+			\item \Tommaso{}.
+		\end{itemize}
+	\end{itemize}
+\section{Ordine del giorno}
+
+\begin{itemize}
+	\item\textbf{Chiarimenti sulle funzionalità aggiuntive e opzionali;}
+	\item\textbf{Discussione sulle API offerte da Zextras Drive;}
+	\item\textbf{Discussione sul linguaggio di programmazione meglio portato per la realizzazione del progetto;}
+	\item\textbf{Progettazione dell'algoritmo di sincronizzazione: modalità da preferire e approccio corretto.}
+
+
+\end{itemize}
+
+\newpage
+
+
+\section{Resoconto}
+La riunione è stata richiesta dal gruppo \textit{\Gruppo{}} per approfondire la conoscenza del referente dell'azienda proponente e per chiarire dubbi sorti dopo una attenta analisi del capitolato.
+Durante l'incontro è stato possibile sottoporre una serie di quesiti, sia sulle funzionalità desiderate dall'azienda e sia su aspetti prettamente tecnici.
+Di seguito è riportata la lista degli argomenti affrontati.
+\subsection{Chiarimenti sulle funzionalità aggiuntive e opzionali}
+Il proponente ha esposto le caratteristiche fondamentali del prodotto da sviluppare, evidenziando l'importanza che riveste la corretta realizzazione dell'algoritmo di sincronizzazione.
+I requisiti fondamentali sono:
+\begin{itemize}
+	\item Sviluppo dell'algoritmo di sincronizzazione;
+	\item Sviluppo di un'interfaccia multipiattaforma;
+	\item Integrazione con il prodotto Zextras Drive.
+\end{itemize}
+Sono inoltre stati discussi i requisiti opzionali.
+\subsection{Discussione sulle API offerte da Zextras Drive}
+L'azienda ha esposto le API offerte dal prodotto Zextras Drive e discusso la loro documentazione con il gruppo. È stato inoltre chiarito che l'azienda potrà aggiungere API se  desiderate dal gruppo e se di possibile realizzazione.
```

Comment:
"se desiderate dal gruppo" suona male. Lo sostituirei con "se di opportuno interesse da parte del gruppo" oppure "se vengono richieste dal gruppo"

Reply:
In realtà ho notato che la frase risultava di difficile comprensione, ho cambiato in " sotto opportuna richiesta da parte del gruppo", che ne pensi?

Reply:
va bene